src/sagemaker/apiutils/_base_types.py

```diff
@@ -42,20 +42,21 @@ def _boto_ignore(cls):
         return ["ResponseMetadata"]
 
     @classmethod
-    def from_boto(cls, boto_dict, **kwargs):
+    def from_boto(cls, boto_dict, **kwargs):  # pylint: disable=R1710
```

Comment:
suggested edit: address this and not ignore?

```
if boto_dict is None:
    return None

...
```

Reply:
sure, done.